added an is_chatting boolean flag to the PlayerState struct

mgaertne opened the pull request:
You can now check a minqlx.Player whether their chat indicator is active via minqlx.Player.state.is_chatting

mgaertne committed:
added an is_chatting boolean flag to the PlayerState struct

You can now check a minqlx.Player whether their chat indicator is active via minqlx.Player.state.is_chatting

em92 commented on python_embed.c:
```diff
@@ -757,7 +758,8 @@ static PyObject* PyMinqlx_PlayerState(PyObject* self, PyObject* args) {
     PyStructSequence_SetItem(state, 11, flight);
 
-    PyStructSequence_SetItem(state, 12, PyBool_FromLong(g_entities[client_id].client->ps.pm_type == 4));
+    PyStructSequence_SetItem(state, 12, PyBool_FromLong(g_entities[client_id].client->ps.eFlags & EF_TALK != 0));
+    PyStructSequence_SetItem(state, 13, PyBool_FromLong(g_entities[client_id].client->ps.pm_type == 4));
```
Could you please set 13 to your new attribute?  For better git blaming (`git blame`)

em92 commented on python_embed.c:
```diff
@@ -117,6 +117,7 @@ static PyStructSequence_Field player_state_fields[] = {
     {"powerups", "The player's powerups."},
     {"holdable", "The player's holdable item."},
     {"flight", "A struct sequence with flight parameters."},
+    {"is_chatting", "Whether the player is currently chatting."},
     {"is_frozen", "Whether the player is frozen(freezetag)."},
```
After you apply my suggestion from https://github.com/MinoMino/minqlx/pull/121/files#r1292673077, this also needs to be changed.

mgaertne committed:
Update python_embed.c

moved the field after the last previous position to keep consistent with previous versions of minqlx.

mgaertne commented:
I think, I'm done here.
